WooCommerce/WooCommerce.xcodeproj/project.pbxproj

```diff
@@ -11036,7 +11040,7 @@
 				ASSETCATALOG_COMPILER_WIDGET_BACKGROUND_COLOR_NAME = WidgetBackground;
 				CLANG_CXX_LANGUAGE_STANDARD = "gnu++17";
 				CLANG_ENABLE_OBJC_WEAK = YES;
-				CODE_SIGN_ENTITLEMENTS = StoreWidgets/Entitlements/StoreWidgets-Debug.entitlements;
+				CODE_SIGN_ENTITLEMENTS = "StoreWidgets/Entitlements/StoreWidgets-Debug.entitlements";
```

Comment:
I am wondering the reason for these changes, adding "" to the entitlements, perhaps the Xcode version?

Reply:
Nothing deliberate, so you're probably right... I'm using 13.4.1 though, which is the same version as CI, so I'm going to keep these changes. StoreWidgets is a new target which may have been added by someone else using a mismatching version of Xcode...

Reply:
Just looping in, I happen to use `13.4.1` as well and the same changes appeared on my [latest PR](https://github.com/woocommerce/woocommerce-ios/pull/7680#discussion_r966814519). I kept them after checking-in with César and from what it looks like, consistency with the rest of sign entitlements 👍